core/platform/search/elastic_search_services.py

```diff
@@ -33,7 +33,9 @@
 # needed, we will have to change this to use the search_after parameter or
 # scrolling option as mentioned in the link above.
 MAXIMUM_NUMBER_OF_RESULTS = 10000
-ES = elasticsearch.Elasticsearch()
+ES = elasticsearch.Elasticsearch([{
+    'host': feconf.ES_HOST, 'port': feconf.ES_PORT}
```

Comment:
Please format as follows:

```
ES = elasticsearch.Elasticsearch([{
    'host': feconf.ES_HOST,
    'port': feconf.ES_PORT
}])
```

(we prefer breaking immediately after brackets/braces, otherwise stuff gets too squeezed to the right).

Reply:
Done